index.html

```diff
@@ -2,23 +2,23 @@
 <html>
 <head>
     <!-- Standard Meta Tags-->
-    <title>BrickHack 7</title>
+    <title>BrickHack Club</title>
```

Comment:
Shouldn't this match the other metadata titles (e.g. Twitter, OpenGraph)?
```suggestion
    <title>BrickHack 8</title>
```

Reply:
Oop yes good catch, I made an earlier version that was for the Club website